modin/pandas/test/test_io.py

```diff
@@ -604,6 +605,13 @@ def test_read_csv_encoding_976(self):
     @pytest.mark.parametrize("encoding", [None, "latin8", "utf16"])
     @pytest.mark.parametrize("engine", [None, "python", "c"])
     def test_read_csv_compression(self, make_csv_file, compression, encoding, engine):
+        if (
+            compression == "zip"
+            and PandasCompatVersion.CURRENT == PandasCompatVersion.PY36
+        ):
+            pytest.xfail(
```

Comment:
it is possible, but it feels more logical to me to "expect to fail" as this is a failure which we might want to (eventually) fix; I treat skips as things which don't make sense in given conditions, like testing how named sockets work on Windows (because Windows doesn't have named sockets)